Sources/TokamakCore/Views/View.swift

```diff
@@ -21,12 +21,14 @@ public protocol View {
   @ViewBuilder var body: Self.Body { get }
 }
 
-extension Never: View {
+extension Never {
   public var body: Never {
     neverBody("Never")
   }
 }
 
+extension Never: View {}
```

Comment:
Just curious, is there any effective difference in how this extension is declared?

Reply:
No, just seemed to better signify that the `body` is shared between `View` and `Scene`. Lmk if you want it changed back.

Reply:
This makes perfect sense, thanks!